### PR TITLE
UI improvements

### LINKED ISSUE #55: [Design] Improve the logo.

### PR DESCRIPTION
**Pull request by sanduninduwara**
Add the main logo and fix styles.
Add conditional rendering for the Navbar.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,12 +1,15 @@
 import { BrowserRouter } from 'react-router-dom';
 import Routes from './routes';
 import ThemeCustomization from './themes';
+import NavBar from './components/Navbar';
 
 // ==============================|| APP - THEME, ROUTER, LOCAL  ||============================== //
 
 const App = () => (
   <ThemeCustomization>
+
```
Comment (chathuranga-jayanath-99) [inline src/App.js]: can remove this new line

File: src/components/Navbar.js [hunk 2]
```diff
@@ -3,17 +3,29 @@ import Toolbar from '@mui/material/Toolbar';
...
   return (
-    <AppBar position="static">
+    <AppBar position="static" sx={{ backgroundColor:"#00417D"}}> 
       <Container maxWidth="xl">
         <Toolbar disableGutters sx={{ display: 'flex', justifyContent: 'space-between' }}>
             <IconButton color="inherit" aria-label="logo" sx={{marginLeft:'30px'}}>
-            <img src="../LogoMain.png" alt="Logo" style={{ height: '160px' }} />
-         </IconButton>
-          <Button variant="contained" color='error'>Log Out</Button>
+            <img src="../LogoMain1.png" alt="Logo" style={{ height: '45px',marginLeft:'30px'}} />
```
Comment (chathuranga-jayanath-99) [inline src/components/Navbar.js]: can indent this line

Comment (Malith-19) [reply src/components/Navbar.js]: Addressed by [00dd588](https://github.com/cse-mosip/Exam-Attendance-UI/pull/57/commits/00dd588c1c722aa5f819c9a11f629962c6ac8e29)

File: src/App.js (hunk 1 above)
Comment (Malith-19) [reply src/App.js]: Addressed by [00dd588](https://github.com/cse-mosip/Exam-Attendance-UI/pull/57/commits/00dd588c1c722aa5f819c9a11f629962c6ac8e29)

File: src/components/Navbar.js [hunk 3]
```diff
@@ -1,19 +1,50 @@
...
+import Toolbar from "@mui/material/Toolbar";
+import IconButton from "@mui/material/IconButton";
+import Container from "@mui/material/Container";
+import Button from "@mui/material/Button";
+import { useLocation, useNavigate } from "react-router-dom";
 
 function NavBar() {
+  const location = useLocation();
+  const currentPath = location.pathname;
+  console.log(currentPath);
```
Comment (jananga99) [inline src/components/Navbar.js]: Please remove unnecessary console logs

**Review by jananga99 — APPROVED** [review]

File: src/components/Navbar.js (hunk 3 above)
Comment (Malith-19) [reply src/components/Navbar.js]: Addressed by [4852eb4](https://github.com/cse-mosip/Exam-Attendance-UI/pull/57/commits/4852eb4d95e7e61b48ddee069e0c27ab6e8929ec)

**Review by chathuranga-jayanath-99 — APPROVED** [review]

**Pull request merged**